src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -338,40 +339,73 @@ export default {
 						visitorId: this.cookieStore.get('uiv') || null
 					},
 				})
-				.then(kivaBraintreeResponse => {
-					// Check for errors in transaction
+				.then(async kivaBraintreeResponse => {
+					// extract transaction saga id or transaction id from response
+					const transactionResult = this.useAsyncCheckout
```

Comment:
The new endpoint also returns a transaction ID? The naming of this as "result" confused me at first, since we used "id" before.

Reply:
Changed to `transactionResponse`